wiki/Submission/zh.md

```diff
@@ -1,27 +1,27 @@
-# 提交
+# 提交谱面
 
-[谱面](/wiki/Beatmap)可以通过[谱面编辑器](/wiki/Beatmap_Editor)提交至 osu! 网站。提交谱面可以使其他用户注意到该谱面，且该谱面可能会标记为 [Ranked](/wiki/Beatmap/Category#ranked) 或 [Loved](/wiki/Beatmap/Category#loved)。
+制作好的[谱面](/wiki/Beatmap)可以通过[谱面编辑器](/wiki/Beatmap_Editor)提交至 osu! 官网上。提交后的谱面就能得到其他玩家的关注，并有机会被标记为 [Ranked](/wiki/Beatmap/Category#ranked) 或 [Loved](/wiki/Beatmap/Category#loved)。
 
-![](img/bss_warning.png "谱面提交系统的截图")
+![](img/bss-warning-ZH.png "谱面提交系统的截图")
 
-在编辑器内的`文件`下拉列表中选择`上传地图...`（快捷键：`Ctrl` + `Shift` + `U`）会打开**谱面提交系统**（***Beatmap Submission System***，简写为***BSS***）窗口。窗口首先会列出帮助用户解决做图相关的问题、寻求谱面的反馈以及确保地图合理的资源。若在使用系统时遇到问题，请参阅指南[如何解决谱面提交系统的问题](/wiki/Guides/BSS_Issues)。
+在编辑器内的`文件`下拉列表中选择`上传地图…`（快捷键：`Ctrl` + `Shift` + `U`）会打开**谱面提交系统**（***Beatmap Submission System***，简写为***BSS***）窗口。窗口中首先会列出一些资源，这些资源可以帮助用户解决与制谱有关的问题、寻求谱面的反馈，并确保谱面符合 Ranked 谱面的标准。若在使用系统时遇到问题，请参阅指南[如何解决谱面提交系统的问题](/wiki/Guides/BSS_Issues)。
 
-如果用户上传的谱面还没有提交至 osu! 网站，窗口会提示用户可以再上传多少张地图。若谱面已被提交，窗口会警告上传地图会使谱面重置。若谱面已成为坟图，窗口会警告谱面将会改变为 pending 状态。
+如果用户上传的谱面集还尚未提交至 osu! 网站，窗口上会显示出用户还能上传的新谱面集数量。若谱面集已被提名，用户会收到“上传地图会使谱面提名状态重置”的警告；若谱面集已成为坟图，用户会收到“谱面将会变为 Pending 状态”的警告。
```

Comment:
这段需要求证一下
因为当时我实验的时候更新坟图并没有给我提示
但是过去确实是有这个提示

It's need to verify.
Because when I did the experiment, updating the graveyard map didn't give me a warning.
But it used to be have this warning.

Reply:
![image](https://user-images.githubusercontent.com/18349191/105871088-ea927f00-6033-11eb-992e-213deda94f89.png)
There's warning.

Reply:
and for qualified map: ![image](https://user-images.githubusercontent.com/18349191/105872022-e155e200-6034-11eb-9db2-0af51b0092e3.png)

image source: [Kuse](https://osu.ppy.sh/users/6984103)'s [2020 Visual Novel Songs Compilation](https://osu.ppy.sh/beatmapsets/1316762) (qualified status at 2021-01-26 16:15 UTC)